AWGL/somatic_db
# Bug fix

## Change description
- Bug fix for issue that occurred during deployment, the wrong PK was used for adding a new poly so it threw a yellow error screen. This wasn't picked up in the test set because the database was small and the variant PK and VariantToVariantList PKs happened to match
- Also committed some changes to tidy up the repo
  - Changed backup script permissions as they were changing whenever we did a git pull, causing the backups to stop
  - Default the settings to the cluster setup, so there's no uncommitted changes on the live db
  - Add unsaved query scripts, one to help with the previous deployment and one to show all variants
  - Add FIGHT-209 panels from https://github.com/AWGL/TSO500_post_processing/pull/52

## Testing description
- Manual check that the error is no longer being seen and check that polys are now added as expected
- Check that panel matches the one from the pipeline
- Check that backups are running (`/data_heath/temp/db_backups/somatic_db/` on Wren and `/home/ew/db_backups` on gen02)
- Query scripts don't require a check

## Test data location
N/A

## Verification results
All checks pass
